package.json

```diff
@@ -2,25 +2,24 @@
   "name": "memize",
   "version": "1.1.0",
   "description": "Unabashedly-barebones memoization library with an aim toward speed",
-  "main": "index.js",
+  "main": "dist/index.cjs",
+  "module": "dist/index.js",
```

Comment:
I'm curious how well `module` is supported outside Webpack & Rollup for things like tree-shaking, and wonder if it might be better to...

1. Drop the `cjs` output altogether and require ESM (breaking change, but that's okay I think?)
2. Or, use the [`package.json` `exports` semantics](https://nodejs.org/api/packages.html#dual-commonjses-module-packages) for better future-compatibility

Reply:
I'm getting errors when running the tests of needing `.mjs` extensions or to use type `module`. 

```
(node:24777) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
(Use `node --trace-warnings ...` to show where the warning was created)
```

When I added the `.mjs` extension to the root `index.js` file with all the `memize` source code it in to `index.mjs` the type declaration ended up being named `index.d.mts` which seems odd.